src/Filesystem/Filesystem.php

```diff
@@ -445,8 +438,10 @@ protected function findSymlinks()
                     continue;
                 }
 
-                // Get subfolders
-                $iterator($directory->getPathname());
+                // Get subfolders if "develop.allowDeepSymlinks" is enabled.
+                if ($deep) {
+                    $iterator($directory->getPathname());
```

Comment:
Don't you need to pass the value of $deep to this iterator call too?

Reply:
@LukeTowers nope - it uses the `$deep` variable from the outer scope. :)

Reply:
Ah, completely missed the use statement there :) Looks good!